src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -234,6 +234,9 @@
       <Name>NServiceBus.AcceptanceTesting</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Dunno, I noticed that too, VS added it, I suppose when I installed the NUnit runner, not sure.